bot/code_review_bot/backend.py

```diff
@@ -121,6 +121,9 @@ def publish_issues(self, issues, revision, mercurial_repository=None, bulk=0):
                         continue
                     valid_data.append((issue, issue.as_dict(issue_hash=issue_hash)))
 
+                if not valid_data:
+                    continue
```

Comment:
Let's add a warning here, I know we have one for "Missing issue hash", but this would be a little different (warning when no issue was valid).